polaris-react/src/components/AlphaCard/AlphaCard.tsx

```diff
@@ -22,8 +25,11 @@ export interface AlphaCardProps {
   background?: CardBackgroundColorTokenScale;
   /** The spacing around the card
    * @default '5'
```

Comment:
I just noticed `Card` has responsive padding (goes down to `4` on small screens, so it probably makes sense to update default to match what `Card` currently does? @sarahill what do you think?

Reply:
True, I think we should add it as a default, I will push up a commit so that it matches `Card` 👍